cert-manager/namespaces/cert-manager/Deployment-cert-manager.yaml

```diff
@@ -1,61 +1,47 @@
-# Source: cert-manager/templates/deployment.yaml
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: cert-manager
-  namespace: "cert-manager"
   labels:
     app: cert-manager
-    app.kubernetes.io/name: cert-manager
+    app.kubernetes.io/component: controller
     app.kubernetes.io/instance: cert-manager
-    app.kubernetes.io/managed-by: Tiller
-    helm.sh/chart: cert-manager-v0.13.1
+    app.kubernetes.io/name: cert-manager
+  name: cert-manager
+  namespace: cert-manager
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: cert-manager
-      app.kubernetes.io/name: cert-manager
+      app.kubernetes.io/component: controller
       app.kubernetes.io/instance: cert-manager
-      app.kubernetes.io/managed-by: Tiller
+      app.kubernetes.io/name: cert-manager
   template:
     metadata:
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: "9402"
+        prometheus.io/scrape: "true"
       labels:
         app: cert-manager
-        app.kubernetes.io/name: cert-manager
+        app.kubernetes.io/component: controller
         app.kubernetes.io/instance: cert-manager
-        app.kubernetes.io/managed-by: Tiller
-        helm.sh/chart: cert-manager-v0.13.1
-      annotations:
-        prometheus.io/path: "/metrics"
-        prometheus.io/scrape: 'true'
-        prometheus.io/port: '9402'
+        app.kubernetes.io/name: cert-manager
     spec:
-      serviceAccountName: cert-manager
       containers:
-        - name: cert-manager
-          image: "quay.io/jetstack/cert-manager-controller:v0.13.1"
-          imagePullPolicy: IfNotPresent
-          args:
-          - --v=2
-          - --cluster-resource-namespace=$(POD_NAMESPACE)
-          - --leader-election-namespace=kube-system
-          - --webhook-namespace=$(POD_NAMESPACE)
-          - --webhook-ca-secret=cert-manager-webhook-ca
-          - --webhook-serving-secret=cert-manager-webhook-tls
-          - --webhook-dns-names=cert-manager-webhook,cert-manager-webhook.cert-manager,cert-manager-webhook.cert-manager.svc
-          - --max-concurrent-challenges=20
-          ports:
-          - containerPort: 9402
-            protocol: TCP
-          env:
-          - name: POD_NAMESPACE
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.namespace
-          resources:
-            requests:
-              cpu: 10m
-              memory: 32Mi
-            
-
+      - args:
+        - --v=2
+        - --cluster-resource-namespace=$(POD_NAMESPACE)
+        - --leader-election-namespace=kube-system
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: quay.io/jetstack/cert-manager-controller:v1.1.0
+        imagePullPolicy: IfNotPresent
+        name: cert-manager
+        ports:
+        - containerPort: 9402
+          protocol: TCP
+        resources: {}
+      serviceAccountName: cert-manager
```

Comment:
This Deployment doesn't needs resources requests anymore? This existed in the original v0.13 manifest (from jetstack repo) but apparently has been dropped.

Reply:
This _should_ have resource requests - perhaps the manifests have been manually patched since then 🤔